src/snowflake/snowpark/dataframe_stat_functions.py

```diff
@@ -42,7 +42,7 @@ def approx_quantile(
 
             >>> df = session.create_dataframe([1, 2, 3, 4, 5, 6, 7, 8, 9, 0], schema=["a"])
             >>> df.stat.approx_quantile("a", [0, 0.1, 0.4, 0.6, 1])
-            [0.5, 1.5, 4.5, 6.5, 9.5]
+            [-0.5, 0.5, 3.5, 5.5, 9.5]
```

Comment:
Curious why is this changed?

Reply:
The original one is not correct https://github.com/snowflakedb/snowpark-python/blob/c838f17c3fb373f7f3cf2192c6a5bf2d74a0dc19/tests/integ/scala/test_dataframe_suite.py#L419